src/crafting_gui.cpp

```diff
@@ -48,6 +51,40 @@ void load_recipe_category( JsonObject &jsobj )
     }
 }
 
+std::string get_cat_name( std::string prefixed_name )
+{
+    if( prefixed_name.find("CC_") == 0 ) {
+        return prefixed_name.substr( 3, prefixed_name.size() - 3 );
+    }
+
+    debugmsg( ( "could not get screen name of: " + prefixed_name ).c_str() );
```

Comment:
This string usage is a bit cumbersome and a bit dangerous. If the prefixed names contains format specifiers (like `%s`), they will be forwarded to `debugmsg` and cause havoc as the matching arguments to fill in are missing.

You can let `debugmsg` do all the string formatting instead:
```C++
debugmsg( "Could not ... of %s", prefixed_name.c_str() );
```
Same for the other call to `debugmsg`.

And the parameters to the functions here should probably be const references.